scripts/initializeDao.ts

```diff
@@ -1,14 +1,57 @@
-import { autocratProgram, initializeDAO } from "./main";
+import { provider, DAO, initializeDAO, BASE_TOKEN, QUOTE_TOKEN } from "./main";
 import * as anchor from "@coral-xyz/anchor";
+// @ts-ignore
+import * as token from "@solana/spl-token";
 
-const { PublicKey, Keypair, SystemProgram } = anchor.web3;
-const { BN, Program } = anchor;
+const { Keypair } = anchor.web3;
 
-const META = new PublicKey("METADDFL6wWMWEoKTFJwcThTbUmtarRJZjRpzUvkxhr");
-const USDC = new PublicKey("EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v");
+const TOKEN = BASE_TOKEN;
+const USDC = QUOTE_TOKEN;
 
 async function main() {
-  await initializeDAO(META, USDC);
+  console.log('Initializing Futarchic DAO')
+  console.log(`Base Token ${BASE_TOKEN.toString()}`)
+  console.log(`Quote Token ${QUOTE_TOKEN.toString()}`)
+  if(DAO.publicKey) {
+    console.log("ALREADY INITIALIZED DAO WITH THIS NAME!!")
+    return;
+  }
+  // Generate KP for use with DAO
+  const daoKP = Keypair.generate();
+
+  // Fetch data about our token...
+  const tokenData = await token.getMint(provider.connection, TOKEN);
+  const decimals = tokenData.decimals;
+  console.log(`Fetched Token Decimals: ${decimals}`)
+
+  // TODO: Query JUP for swap amount to get value
+  const tokenPrice = 0.0008 // Rounded to nearest
+
+  // Get a price between 0 and 10 whole value
+  // Example: 1 META @ $400 would be 0.01 as that's $4
+  // Example: 1 DEAN @ $0.000795 would be 10_000 as that's $7.81
+  const lotOfToken = 10_000 // This is token quantity to get between $0-$10
```

Comment:
This should be calculated, but I've exhausted my capacity to think / understand what would be best.